forge-game/src/main/java/forge/game/GameActionUtil.java

```diff
@@ -270,35 +273,6 @@ public static final List<SpellAbility> getAlternativeCosts(final SpellAbility sa
                     foretold.putParam("AfterDescription", "(Foretold)");
                     alternatives.add(foretold);
                 }
-
-                // some needs to check after ability was put on the stack
-                // Currently this is only checked for Toolbox and that only cares about creature spells
-                if (source.isCreature() && game.getAction().hasStaticAbilityAffectingZone(ZoneType.Stack, StaticAbilityLayer.ABILITIES)) {
-                    Zone oldZone = source.getLastKnownZone();
-                    Card blitzCopy = source;
-                    if (!source.isLKI()) {
-                        blitzCopy = CardUtil.getLKICopy(source);
-                    }
-                    blitzCopy.setLastKnownZone(game.getStackZone());
-                    lkicheck = true;
-
-                    blitzCopy.clearStaticChangedCardKeywords(false);
-                    CardCollection preList = new CardCollection(blitzCopy);
-                    game.getAction().checkStaticAbilities(false, Sets.newHashSet(blitzCopy), preList);
-
-                    // currently only for Keyword BLitz, but should affect Dash probably too
-                    for (final KeywordInterface inst : blitzCopy.getKeywords(Keyword.BLITZ)) {
-                        // TODO with mana value 4 or greater has blitz.
-                        for (SpellAbility iSa : inst.getAbilities()) {
-                            // do only non intrinsic
-                            if (!iSa.isIntrinsic()) {
-                                alternatives.add(iSa);
-                            }
-                        }
-                    }
-                    // need to reset to Old Zone, or canPlay would fail
-                    blitzCopy.setLastKnownZone(oldZone);
-                }
```

Comment:
This logic was moved, mostly unchanged, into `getAlternativeCostsGrantedByStaticAbilities`.